scio-avro/src/main/scala/com/spotify/scio/avro/types/AvroType.scala

```diff
@@ -125,6 +125,32 @@ object AvroType {
   }
 
   /**
+    * Macro annotation for a file which contains Avro schema.
+    *
+    * Generate case classes for an Avro schema. File can be either local or remote files.
+    * For example file can be located on Google Cloud Storage:
+    *
+    * {{{
+    *   @AvroType.fromFile("gs://myBucket/myFolder/schema-file.avsc")
+    *   class MyRecord
+    * }}}
+    *
+    * For local files, you need to either provide absolute path,
+    * or path relative to project root directory. For example:
+    *
+    * {{{
+    *   @AvroType.fromFile("sub-project/src/main/avro/schema-file.avsc")
+    *   class MyRecord
+    * }}}
+    *
+    * Also generate a companion object with convenience methods.
+    * @group annotation
+    */
+  class fromFile(schemaFile: String) extends StaticAnnotation {
```

Comment:
`fromFile` can be a bit confusing with `fromPath` though, which could also take a single file path. `fromSchemaFile` instead?